app/src/main/java/fi/helsinki/opintoni/service/OfficeHoursService.java

```diff
@@ -69,10 +68,9 @@ private int compareNames(String name1,String name2) {
     }
 
     private static String convertToSortableName(String name){
-        ArrayList<String> nameParts = new ArrayList<>(Arrays.asList(name.trim().split(" ")));
-        String surname = nameParts.remove(nameParts.size() - 1);
-        nameParts.add(0, surname);
-        return nameParts.stream().collect(Collectors.joining(""));
+        List<String> nameParts = new ArrayList<>(Arrays.asList(name.trim().split(" ")));
```

Comment:
`new ArrayList`in käyttö turhaa -- riittää sanoa `List<String> nameParts = Arrays.asList(name.trim().split(" "))` (`Arrays.asList` palauttaa listan)